### DISCUSSION TITLE
Welcome to EasyUpiPayment-Android Discussions!

### DISCUSSION THREAD

**Discussion by PatilShreyas** [PatilShreyas/EasyUpiPayment-Android#82] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by shivdattsingh** [comment]
I want to upi integration in my android app Please contact me -- sundareducation@gmail.com

Android app developer can contact

**Reply by PatilShreyas** [reply]
You can refer to the docs: http://patilshreyas.github.io/EasyUpiPayment-Android

**Comment by gangadharKorrapati** [comment]
off topic but a related a question as I have just read the UPI linking spec.
I think https://upigateway.com/ and https://www.bharatupi.com/ are generating qr codes for payments using [upi linking spec](https://web.archive.org/web/20200921110005/https://www.npci.org.in/sites/default/files/UPI%20Linking%20Specs_ver%201.6.pdf) and are using android notifications and transaction messages on merchant's mobile for status of transaction and are posting transaction status to their server which in turn responds to their websdk. Are my assumptions correct?

**Comment by deepsinghrathi** [comment]
Sir today my one of Clint give me video about upi transection in which he made transection and then by dial wrong pin there transection goes failed but your easyupipayment give success in my system. Sir please check that issue first because my app is live and in my app payment method of upi is widely used.
